app.py

```diff
@@ -223,6 +223,17 @@ def populate_shotgun(self, session_id, info):
         
         # done!
         self._submission_done = True
+
+        # Log usage metrics
+        try:
+            self.log_metric("Version Create")
```

Comment:
I think perhaps `Sequence Export`. That would log as `(tk-flame) tk-flame-review Sequence Export`. I'll update.